ui/src/submissions/literature/components/ArticleForm.jsx

```diff
@@ -65,7 +66,7 @@ class ArticleForm extends Component {
             />
             <Field name="volume" label="Volume" component={TextField} />
             <Field name="issue" label="Issue" component={TextField} />
-            <Field name="year" label="Year" component={TextField} />
+            <Field name="year" label="Year" component={NumberField} />
```

Comment:
do we have a tests for these value types?

Reply:
tested via `e2e`, test is probably not filling the year field.

Reply:
it's visible in screenshot though, that year is a number field.

Reply:
can we add the year field?